cmd/dockerd/daemon.go

```diff
@@ -469,6 +470,18 @@ func loadDaemonCliConfig(opts *daemonOptions) (*config.Config, error) {
 	return conf, nil
 }
 
+func warnOnDeprecatedConfigOptions(config *config.Config) {
```

Comment:
This function should also handle  `--restart`?

Reply:
It is mostly intended for the options set in the daemon configuration file. Can't find if --restart has an equivalent in the daemon config file.